### DISCUSSION TITLE
No access to 1Wallet Authenticator

### DISCUSSION THREAD

**Discussion by pjconleyy** [polymorpher/one-wallet#236] [General]
I just got a new phone and verizon took my old one. When I transferred everything from my old phone to my new one, it did not keep the authenticator for my 1wallet. As of right now I am in my 1wallet but I cannot use the ONE in there because I need the authenticator codes.
My wallet address is one1qw8x5ekv4nhyhyn996e97s6q3c2t2gqhe9cpkj

**Comment by polymorpher** [comment]
This is being discussed on Telegram. The process of recovery is underway.
